docs/ldml/tr35-keyboards.md

```diff
@@ -1316,11 +1324,16 @@ _Attribute:_ `form` (required)
 > This attribute specifies the physical layout of a hardware keyboard,
 > or that the form is a `touch` layout.
 >
-> It is recommended to always have at least one hardware (non-touch) form.
+> When using an on-screen keyboard, if there is not a `<layers form="touch">`
+> element, the hardware elements can be used for on-screen use. 
```

Comment:
This is a little ambiguous. On-screen keyboard may refer to a visual representation of a hardware keyboard or a touch layout. Typically, the implementation would use the current hardware layout if it is intended to be a visual representation for user reference.

Something like:
```suggestion
> When using an on-screen touch keyboard, if the keyboard does not specify a `<layers form="touch">`
> element, a `<layers form="{hardware}">` element can be used as an fallback alternative. 
```

Reply:
Makes sense. and this is certainly a common use case.